baselines/flwr_baselines/publications/leaf/femnist/constants.py

```diff
@@ -0,0 +1,4 @@
+import torch
+
+RANDOM_SEED = 42
+DEVICE = torch.device("cpu")
```

Comment:
Those should probably be set in the hydra configs.

Reply:
I added them to the config now (and generally changed the structure of the config)